### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by alejandrogallo** [papis/papis#461] [Ideas]
After 6 years, I open this discussion again.

I'd like to ask for logos, I was toying a while ago with an idea and I came up with this, but I'm very bad at these things

![image](https://user-images.githubusercontent.com/12715358/209392351-fd374f2f-b563-4ac0-ad13-0430b81256b2.png)

But maybe and hopefully someone is more gifted ... ;)

**Comment by jghauser** [comment]
A close friend of mine works in advertising and has made logos in the past. I think he would probably happy to make something simple for us if I ask him. Maybe if people can give me some input about topic/content/style/vibe, I can pass this on and ask him to draft something. What do you think?

With regard to your proposal, I think it looks pretty but I don't really understand what it is supposed to be. I think something that instantly gets across the sciency/referency/researchy point of papis would be better suited to becoming recognisable.

**Reply by alexfikl** [reply]
That would be awesome! I have a graphic designer friend and will see if he has some time to play around with this too :D

Otherwise, https://github.com/nschloe has some nice clean logos for his science-y projects that could serve as inspiration.

**Reply by alejandrogallo** [reply]
I also have some friends like that, but somehow they (understandably) never manage to find the time
to sit down and think about the logo. This logo I don't know what it signifies, but I like geometry and so and they kind of look like two papyrus? :D I'd rather have someone else doing the logo of course, maybe your friends have some time over the holidays and they want to contribute to an FOSS project ;)

**Reply by jghauser** [reply]
I'm gonna ask my friend in the next days and we'll see if they're motivated haha

**Comment by jghauser** [comment]
So I talked to my friend last week and he said he'd be happy to create a logo for us. He'll get back to me next week with a plan :grinning:

EDIT: Oops, that was meant to be a response to #40. but it fits here too!
